tests/test_reprocessVisitImage.py

```diff
@@ -0,0 +1,83 @@
+# This file is part of ci_hsc_gen3.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import os
+import unittest
+
+import lsst.utils.tests
+import numpy as np
+from lsst.daf.butler import Butler
+from lsst.utils import getPackageDir
+
+
+class TestReprocessVisitImageOutputs(lsst.utils.tests.TestCase):
+    """Test the output data products of reprocessVisitImage task make sense.
+
+    This is a regression test and not intended for scientific validation.
+    """
+
+    def setUp(self):
+        self.butler = Butler(os.path.join(getPackageDir("ci_imsim"), "DATA"),
+                             writeable=False, collections=["LSSTCam-imSim/runs/ci_imsim"])
+        self.dataId = {"detector": 55, "visit": 206039, "band": "y"}
+        self.exposure = self.butler.get("pvi", self.dataId)
+        self.catalog = self.butler.get("sources_footprints_detector", self.dataId)
+
+    def testLocalPhotoCalibColumns(self):
```

Comment:
I do hope that we can remove `LocalPhotoCalib` from the catalogs in the future, since they'll have calibrated values. Not sure if that's ticketed?

Reply:
The `transform*Table` catalog implementations are pandas-heavy and one of the bits of the stack I'm least familiar with, and that makes me really not want to wade in to them as long as they're working.  So if we ever do fix this, I think it'll be because we decide to do a full rewrite of that stuff, and that won't benefit much from a more targeted ticket.